### PR TITLE
Task with the two counters

### PR DESCRIPTION
**Pull request by KrasimirRaikov**
When the smaller one ends it stops the bigger one

### CODE REVIEW

File: src/multi/counter/MultiCounter.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+/**
+ * @author Krasimir Raikov(raikov.krasimir@gmail.com)
+ */
+public class MultiCounter extends Thread {
+
+    private int beginning;
+    private int end;
+    private Thread thread;
+    private boolean flag;
```
Comment (mgenov) [inline src/multi/counter/MultiCounter.java]: I'm not sure that you need this flag. You can check whether the target thread is null.

**Comment by KrasimirRaikov** [comment]
I removed the flag

**Comment by mgenov** [comment]
LGTM ! 

btw, from my point of view, it's more readable with using of break: 

```java
  for (int i = beginning; i <= end; i++) {
    try {
      Thread.sleep(1000);
    } catch (InterruptedException e) {
      break;
    }    
    System.out.println(name + ": " + i);  
```

As you can see, the interrupted exception is handled by breaking the for loop.

**Comment by KrasimirRaikov** [comment]
I agree, so I made the changes

**Comment by mgenov** [comment]
:+1:

**Pull request merged**